pkg/syncer/metadatasyncer.go

```diff
@@ -2805,14 +2805,29 @@ func createStoragePolicyUsageCR(ctx context.Context, quotaClient client.Client,
 			ResourceExtensionName: extensionName,
 		},
 	}
-	err := quotaClient.Create(ctx, &newUsageInstance, &client.CreateOptions{})
+	retryBackoff := wait.Backoff{
+		Duration: 1 * time.Second,
+		Factor:   2,
+		Steps:    50,
+		Jitter:   0.2,
+	}
+	ctxWithTimeout, _ := context.WithTimeout(context.Background(), time.Duration(time.Minute*5))
+	err := wait.ExponentialBackoffWithContext(ctxWithTimeout, retryBackoff, func(_ context.Context) (bool, error) {
+		policyUsageCRCreateErr := quotaClient.Create(ctx, &newUsageInstance, &client.CreateOptions{})
+		if policyUsageCRCreateErr != nil {
+			log.Errorf("Failed to create StoragePolicyUsage for policyID %v storageclass %v resourceKind %v. Err: %+v",
```

Comment:
Can we use %q instead of %v for strings in this log line?

Reply:
fixed